entity-framework/efcore-and-ef6/porting/index.md

```diff
@@ -1,61 +1,71 @@
 ---
-title: Porting from EF6 to EF Core - EF
-description: General information on porting an application from Entity Framework 6 to Entity Framework Core
-author: ajcvickers
-ms.date: 10/27/2016
+title: Port from EF6 to EF Core - EF
+description: A detailed guide to port your 
+author: jeremylikness
+ms.alias: jeliknes
+ms.date: 10/25/2021
 uid: efcore-and-ef6/porting/index
 ---
-# Porting from EF6 to EF Core
+# Port from EF6 to EF Core
 
-Because of the fundamental changes in EF Core we do not recommend attempting to move an EF6 application to EF Core unless you have a compelling reason to make the change.
-You should view the move from EF6 to EF Core as a port rather than an upgrade.
+Entity Framework Core, or EF Core for short, is a total rewrite of Entity Framework for modern application architectures. Due to fundamental changes, there is not a direct upgrade path. The purpose of this documentation is to provide an end-to-end guide for porting your EF6 applications to EF Core.
 
 > [!IMPORTANT]
 > Before you start the porting process it is important to validate that EF Core meets the data access requirements for your application.
 
-## Missing features
+## When to port
 
-Make sure that EF Core has all the features you need to use in your application. See [Feature Comparison](xref:efcore-and-ef6/index) for a detailed comparison of how the feature set in EF Core compares to EF6. If any required features are missing, ensure that you can compensate for the lack of these features before porting to EF Core.
+It is not a requirement that you port your code to EF Core. There are many reasons why you may choose to remain on EF6. The latest version of EF6 supports .NET Core. If your application is running fine and you don't need any of the latest features, you may wish to consider remaining on EF6. Some reasons you might consider porting:
```

Comment:
I'm never sure these days how to unambiguously refer to what used to be .NET Core. Since .NET 5 and .NET 6 are not ".NET Core", then it might be confusing to just say .NET Core here. Maybe explicitly call this out in some way? (Although everything I can think of sounds clumsy! Why did we rename again?)